### PR TITLE
Add docker buildx support for multi-arch images

### PR DESCRIPTION
**Pull request by shahidhs-ibm**
Signed-off-by: Shahid Shaikh <shahid@us.ibm.com>

Update GitHub Actions for `docker buildx` to support multi-arch docker images.

### CODE REVIEW

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -23,9 +23,15 @@ jobs:
...
+      with:
+        buildx-version: latest
+    - name: Docker prepare
+      run: docker image prune -a -f
     - name: Docker Login
       run: docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
-    - name: Push the Docker image
-      run: docker push ${IMAGE_FULL}
+    - name: Build and push the Docker image(s)
+      run: docker buildx build --platform linux/amd64,linux/s390x --cache-from="type=registry,ref=${CACHE_IMAGE_FULL}" --cache-to="type=registry,ref=${CACHE_IMAGE_FULL},mode=max" -t ${IMAGE_FULL}  -f apache.Dockerfile --push .
```
Comment (olexii4) [inline .github/workflows/dockerimage.yml:38]: @shahidhs-ibm  It is not clear for me. 
I think we should add the **CACHE_IMAGE_FULL** into **env**.
Something like this:

```
    env:
      IMAGE_FULL: quay.io/che-incubator/che-dashboard-next:next
      CACHE_IMAGE_FULL: docker.io/cheincubator/che-dashboard-next:cache
```

and
```
steps:
   ...
    - name: "Docker docker.io Login"
      run:  docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" -p "${{ secrets.DOCKERHUB_PASSWORD }}" docker.io
```

Am I right?

Comment (shahidhs-ibm) [reply .github/workflows/dockerimage.yml:38]: @olexii4 Thank you for the review. Nice catch!! :)
So we have option to add CACHE_IMAGE_FULL into env or to remove them completely and use the default values. I am OK with both. Your suggestion?

Comment (olexii4) [reply .github/workflows/dockerimage.yml:38]: I will prefer to add the CACHE_IMAGE_FULL as in the example.

Comment (shahidhs-ibm) [reply .github/workflows/dockerimage.yml:38]: @olexii4 Incorporated the code review comments. Please check.

**Review by olexii4 — APPROVED** [review]

**Pull request merged**